src/ServiceControl.Transports.Tests/ServiceControlMonitoringEndpointTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace ServiceControl.Transport.Tests
+{
+    using System.Collections.Generic;
+    using System.Security.Principal;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+    using ServiceControl.Transports;
+
+    class ServiceControlMonitoringEndpointTests : TransportTestFixture
+    {
+        [Test]
+        public async Task Should_configure_monitoring_endpoint()
+        {
+            var transportSettings = new TransportSettings
+            {
+                ConnectionString = configuration.ConnectionString,
+                MaxConcurrency = 1
+            };
+
+            var endpointName = NServiceBus.AcceptanceTesting.Customization.Conventions.EndpointNamingConvention(typeof(ServiceControlEndpoint));
+            var username = WindowsIdentity.GetCurrent().Name;
+
+            await ProvisionQueues(username, endpointName, "InstanceName.Errors", new List<string>());
+
+            var ctx = await Scenario.Define<Context>()
+                .WithEndpoint<ServiceControlEndpoint>(c => c.CustomConfig(ec =>
+                {
+                    configuration.TransportCustomization.CustomizeMonitoringEndpoint(ec, transportSettings);
+                }))
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            Assert.IsTrue(ctx.EndpointsStarted);
```

Comment:
I guess we could do a send local to make sure it's actually working if we want to be more thorough?